src/ipyvizzustory/storylib/story.py

```diff
@@ -132,22 +132,43 @@ def add_step(self, step: Step) -> None:
 class StorySize:
     """A class for representing the size of a presentation story."""
 
-    def __init__(self, width: Optional[str] = None, height: Optional[str] = None):
+    def __init__(
+        self,
+        width: Optional[Union[int, str]] = None,
```

Comment:
Can width and height be float?

Reply:
fixed in [f536d86](https://github.com/vizzuhq/ipyvizzu-story/pull/75/commits/f536d86a8f4cfeff4b91072378b39bd1cb4c59d6)

Reply:
https://github.com/vizzuhq/ipyvizzu-story/commit/f536d86a8f4cfeff4b91072378b39bd1cb4c59d6#r126048489